### DISCUSSION TITLE
Negative Dimensions Issue

### DISCUSSION THREAD

**Discussion by ghost** [JCBrouwer/maua-stylegan2#23] [Q&A]
I am trying to work through the tauceti example. The first issue I had was that "cyphept-multicolor-latents.npy" is not in the workspace folder. I generated my own latents.npy file, but then I got this error:
The expanded size of the tensor (1800) must match the existing size (10186) at non-singleton dimension 0.  Target sizes: [1800, 9, 512].  Tensor sizes: [10186, 9, 512]

I was able to work around this issue by adjusting the size of the tensor shape to [1800, 9, 512]. Now I am getting the following error:

"ValueError: negative dimensions are not allowed"

![neg_dim](https://user-images.githubusercontent.com/16939652/121376764-66b4a180-c907-11eb-91c4-f826f4fbc911.PNG)

**Comment by JCBrouwer** [comment]
The translation effect in this example is hard-coded to only occur during a specific section of the song. I think you just need to adjust [these two lines](https://github.com/JCBrouwer/maua-stylegan2/blob/master/audioreactive/examples/tauceti.py#L103)) to be within your interpolation. 5591 is the total length in frames of the video (should be 1800 in your case), and 45 and 135 are the timestamps in seconds between which the scrolling effect will take place.

**Reply by ghost** [reply]
That was it. Thank you! I might end up making a notebook with some of these more complex examples. If I do, I will share.
